ui/actions/nuxeo-delete-blob-button.js

```diff
@@ -150,15 +155,65 @@ import '../nuxeo-button-styles.js';
       };
     }
 
+    execute(batchId) {
+      return this.$.nx.connect().then(this._send(batchId));
+    }
+
+    _send(batchId) {
+      let url = [this.$.nx.client._baseURL, 'api/v1/upload', this.index, batchId].join('/');
+      url = url.replace(/(^\/+)|([^:])\/\/+/g, '$2/');
+      url += `/0`;
```

Comment:
This should be not  hardcoded since `/0` => `{fileId}` can be dynamic.
see below API reference.  
https://jira.nuxeo.com/browse/ELEMENTS-1629?focusedCommentId=680011&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-680011

Reply:
done